tests/python/pants_test/interpreter_selection_utils.py

```diff
@@ -36,5 +36,6 @@
 
 deprecated_module(
   removal_version="1.25.0.dev0",
```

Comment:
Can you please change these `removal_versions` to 1.26.0.dev2? We never released a wheel with this deprecation until now, so essentially the deprecation cycle is starting as new here.

Reply:
Done comprehensively.

Reply:
> Done comprehensively.

Great. Thank you for taking the time to do this!